src/env.h

```diff
@@ -713,7 +713,7 @@ class Environment {
   std::map<std::string, uint64_t> performance_marks_;
 
 #if HAVE_INSPECTOR
-  inspector::Agent inspector_agent_;
+  inspector::Agent* inspector_agent_ = nullptr;
```

Comment:
I'd make this `inspector::Agent* const inspector_agent_;`, that way the compiler will complain if someone forgets to assign it in the constructor.

Reply:
Done.